### ISSUE TITLE
Bias?

### ISSUE THREAD

**Issue by fauks** [kaktus85/MightyWattR3#2]
Hello,

I have been using your MightyWatt R3 project to learn how to create my own version. 

I am wondering about the "bias" net. Is this a 2.5v or 5v output from the AD5691? It looks like it is fed into the TLV2372 opamp directly after a resistor divider, and the output is the "bias" net. I also see there is a pad on the rear end of the board for "bias" but I'm not sure what it is used for. 

Is this correct? Also, if there is a better way to communicate with you, please let me know. 

Thank you!

**Comment by kaktus85** [comment]
Hi, I am no longer involved in hardware development but as far as I
remember, the bias was around 75 mV, derived from the 2.5V reference of the
ADC (via R23/R24 divider). The purpose was to offset signals from ground to
have a linear response. Even rail-to-rail op-amps are not completely linear
at 0 V. But they can be at 75 mV.

po 1. 4. 2024 v 3:28 odesílatel DBC ***@***.***> napsal:

**Comment by fauks** [comment]
Thank you! I understand now. I have been beating my head trying to figure out what was going on with the second opamp. I'm learning slowly. 

I plan on scaling this up a little and implementing more modern hardware.

**Comment by fauks** [comment]
One last question, was this designed with the 3.3v logic boards in mind? I thought that is what the VCCIO VREF pin on the shield was used for. I am going to implement a ESP32 and have to figure out what needs to be changed.

**Comment by kaktus85** [comment]
Yes, it works with both 3V3 and 5V boards. You are right, the VCCIO / IOREF
is exactly that. Voltage at this pin is taken as the digital reference
voltage.
But the board needs 5V supply to power the fan, relay and DC/DC converter
that makes higher voltage for the main transistor.

po 1. 4. 2024 v 14:12 odesílatel DBC ***@***.***> napsal:
